tests/cases/conformance/async/es2017/asyncMultiFile_es2017.ts

```diff
@@ -0,0 +1,5 @@
+// @target: es2017

```

Comment:
This test isn't really needed as it was added only to ensure the `__awaiter` helper was added to the correct file.